test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -667,13 +667,17 @@ Describe "Console host api tests" -Tag CI {
 Describe "Pwsh exe resources tests" -Tag CI {
     It "Resource strings are embedded in the executable" -Skip:(!$IsWindows) {
         $pwsh = Get-Item -Path "$PSHOME\pwsh.exe"
-        $pwsh.VersionInfo.FileVersion | Should -BeExactly $PSVersionTable.PSVersion
+        $fileVersion = $PSVersionTable.PSVersion.ToString()
+        if ($fileVersion.Contains("-")) {
+            $fileVersion = $fileVersion.Split("-")[0]
+        }
```

Comment:
@SteveL-MSFT I'm curious why this PR causes this test to fail? Can you please help me understand?